src/helpers/pivot/pivot_menu_items.ts

```diff
@@ -15,3 +15,30 @@ export const pivotProperties: ActionSpec = {
   },
   icon: "o-spreadsheet-Icon.PIVOT",
 };
+
+export const FIX_FORMULAS: ActionSpec = {
+  name: _t("Fix pivot formulas"),
```

Comment:
Is it a magic button to fix all bugs in pivot formulas !!!??
That's so niiiice :astonished:  :upside_down_face:

Reply:
François will have to find a good name :D